app/view_models/admin/auction_show_view_model.rb

```diff
@@ -14,15 +14,6 @@ def csv_report_partial
     end
   end
 
-  def admin_auction_status_partial
```

Comment:
disclaimer: because we are removing this, we are showing statuses for all auctions at all states now in the admin interface. We need to fill some gaps with that (eg: right now, if an "other pcard" auction is pending delivery, we show its c2 status...which makes no sense) but should happen soon (created cards for the cases)